src/main/java/mail/GeneratorPassword.java

```diff
@@ -0,0 +1,22 @@
+package mail;
+
+import java.util.Random;
+
+public class GeneratorPassword {
+
+    public String generatedPassword() {
+
+        String upperWords = "QWERTYUIOPASDFGHJKL";
+        String lower = "qwertyuiopasdfghjkl";
+        String num = "0123456789";
+        String specialChars = "{}*&^%#@:<>";
+        String combination = upperWords + lower + num + specialChars;
```

Comment:
combination is bad name, example:

`validChars`
`validSymbols`
`allowedChars`
`allowedSymbols`